app/helpers/container_helper/textual_summary.rb

```diff
@@ -109,4 +109,23 @@ def textual_se_linux_level
   def textual_run_as_non_root
     {:label => "Run As Non Root", :value => @record.container_definition.run_as_non_root} unless @record.container_definition.run_as_non_root.nil?
   end
+
+  def textual_group_env
+    {
+      :additional_table_class => "table-fixed",
+      :expandable             => true,
+      :labels                 => [_("Name"), _("Type"), _("value")],
+      :values                 => collect_env
+    }
+  end
+
+  def collect_env
+    @record.container_definition.container_env_vars.collect do |var|
+      [
+        var.name,
+        var.value.nil? ? "REFERENCE" : "VALUE",
```

Comment:
since we are using names different from k8s' valueFrom and fieldRef I can't think of a way